packages/common/container-definitions/package.json

```diff
@@ -62,6 +62,35 @@
 		"typescript": "~4.5.5"
 	},
 	"typeValidation": {
-		"broken": {}
+		"broken": {
+			"InterfaceDeclaration_ContainerWarning": {
```

Comment:
Are these still required despite the re-exports in container-definitions?

Reply:
Some of them are!

Reply:
I removed the validation and the entries here and regenerated again, but looks like these ones are required.